arcgis-ios-sdk-samples/Layers/ArcGIS vector tiled layer (custom style)/README.md

```diff
@@ -15,14 +15,24 @@ Pan and zoom to explore the vector tile basemap.
 
 ## How it works
 
-1. Construct an `AGSArcGISVectorTiledLayer` with the URL of a custom style.
+1. Construct an `AGSArcGISVectorTiledLayer` with the URL of a custom style from AGOL.
+    * Follow these steps to create a vector tiled layer with a custom style from offline resources:
+    1. Construct an `AGSVectorTileCache` using the name of the local vector tile package.
```

Comment:
I feel using a different scheme like a,b,c or i,ii,iii would look better a sublist here

Reply:
Haha Markdown does render as i, ii, iii 👇 even it is written as 1, 2, 3... 😁 
![image](https://user-images.githubusercontent.com/9660181/99736963-2c270c80-2a7c-11eb-98d6-7754b3243d56.png)

Reply:
Ohh... that is weird... I haven't check on git markdown, but the readme page in our viewer renders like this

![IMG_838FAB48DA4B-1](https://user-images.githubusercontent.com/8692125/99738333-1a933400-2a7f-11eb-8306-da38504e0226.jpeg)

Reply:
> Ohh... that is weird... I haven't check on git markdown, but the readme page in our viewer renders like this

I see. Yeah apparently GitHub is using a more advanced Markdown parser... 🤥 
On our developer site it also doesn't render correctly as secondary numbered list style. i.e. https://github.com/Esri/arcgis-runtime-samples-ios/pull/981#discussion_r527280529

So yeah maybe consider replacing them with unordered list asterisk, or manually do a, b, c etc.

Reply:
I manually made the roman numerals. Please let me know what you think, thanks!